.travis.yml

```diff
@@ -11,11 +11,6 @@ branches:
 # by .swift-version, unless SWIFT_SNAPSHOT is specified.
 matrix:
   include:
-    - os: linux
```

Comment:
Are you needing to have Swift > 4.0.3? In that case, it would be better to enclose your code with #if/#else rather than removing the whole tests suite.

Reply:
Just trying to fix travis timeout failure.  Restoring code.

Reply:
Travis simply will not complete testing on older versions of Mac OS.  They need to be removed.